### ISSUE TITLE
Reverting/Reapplying patches

### ISSUE THREAD

**Issue by ghost** [cursey/kanan#28]
Would it be possible to add a way to revert and reapply patches?
Say like a command in the command prompt that kanan opens?
e.g. `revert NPCFastText`

I would think something along the lines of the `patch()` method will save the original bytes before changing it. Then the `revert` command will look for changes related to the patch and go back and apply the original bytes.

Then reapplying would basically be rerunning the original script like it is done now, but on-demand.

Or whatever works. I have no idea. :)

**Comment by cursey** [comment]
Yeah I was thinking of adding a `--interactive` mode.  I'll come up with something.

**Comment by cursey** [comment]
I have added most of the groundwork to implement something like this, I'll get around to it soon.
